openslides/users/models.py

```diff
@@ -14,20 +17,96 @@
 from openslides.utils.models import AbsoluteUrlMixin
 from openslides.utils.rest_api import RESTModelMixin
 
+from .exceptions import UserError
+
 
 class UserManager(BaseUserManager):
     """
     UserManager that creates new users only with a password and a username.
     """
-
     def create_user(self, username, password, **kwargs):
         user = self.model(username=username, **kwargs)
         user.set_password(password)
         user.save(using=self._db)
         return user
 
+    def create_or_reset_admin_user(self):
+        """
+        Creates an user with the username admin. If such a user exists, resets
+        it. The password is (re)set to 'admin'. The user becomes member of the
+        group 'Staff' (pk=4).
+        """
+        try:
+            admin = self.get(username='admin')
+        except self.model.DoesNotExist:
+            admin = self.model()
+            admin.username = 'admin'
+            admin.last_name = 'Administrator'
+            created = True
+        else:
+            created = False
+        admin.default_password = 'admin'
+        admin.password = make_password(admin.default_password, '', 'md5')
+
+        try:
+            staff = Group.objects.get(pk=4)
+        except Group.DoesNotExist:
+            if created:
+                raise UserError("Admin user can not be created because the "
+                                "group 'Staff' is not available.")
+            else:
+                raise UserError("Admin user can not be reset because the group "
+                                "'Staff' is not available.")
```

Comment:
Hm. That would be difficult because we don't have a default create_staff_function. And then we need to check Registered group too, because an admin needs more than just some manage permissions. At the moment I don't have enough mood to implement all this logic. You can open an issue if you like.